### DISCUSSION TITLE
Can't see my collaboration projects in the project selection menu when creating a new project in code magic

### DISCUSSION THREAD

**Discussion by BelalNayzak** [codemagic-ci-cd/codemagic-docs#1626] [Q&A] [answered]
I can't see my collabration projects when selecting a github project (i can see only the projects I've created).

**Comment by rafeysup93** [comment, marked as answer]
The collaborators won't be able to list projects as their own in the dropdown. So, what you are seeing is expected behavior.

**Reply by BelalNayzak** [reply]
Alright, got it.
Thanks

**Reply by rafeysup93** [reply]
You're welcome.
